remix-tests/src/runTestFiles.ts

```diff
@@ -44,7 +53,13 @@ export function runTestFiles(filepath: string, isDirectory: boolean, web3: Web3,
         function compile(next: Function) {
             compileFileOrFiles(filepath, isDirectory, { accounts }, next)
         },
-        function deployAllContracts (compilationResult, next: Function) {
+        function deployAllContracts (compilationResult, asts, next: Function) {
```

Comment:
Is there any tupes for result & AST ? I've some types [here](https://github.com/ethereum/remix-plugin/blob/master/projects/utils/src/api/compiler/type/output.ts) for compilation output. I don't know if it can help.